offline/docs.md

```diff
@@ -95,7 +95,7 @@ The following artifacts are provided:
  - `containers-other.tar`
    These are other container images, not deployed inside k8s. Currently, only
    contains Restund.
- - `debs.tar`
+ - `debs-bionic.tar`
```

Comment:
Should this be something like `debs-*.tar` or `debs-(bionic|jammy).tar`?

Reply:
this file is ubuntu 18 specific and the artifact used by those deployments still has the debs.tar file